modules/snappy/1.1.8/patches/add_build_file.patch

```diff
@@ -0,0 +1,95 @@
+--- /dev/null
++++ BUILD.bazel
+@@ -0,0 +1,92 @@
++package(
++    default_visibility = ["//visibility:public"],
++    features = ["header_modules"],
++)
++
++licenses(["notice"])
++
++cc_library(
++    name = "snappy",
++    srcs = [
++        "config.h",
++        "snappy.cc",
++        "snappy-internal.h",
++        "snappy-sinksource.cc",
++        "snappy-stubs-internal.cc",
++        "snappy-stubs-internal.h",
++        "snappy-stubs-public.h",
++    ],
++    hdrs = [
++        "snappy.h",
++        "snappy-sinksource.h",
++    ],
++    copts = [
++        "-DHAVE_CONFIG_H",
++        "-Wno-sign-compare",
++    ],
++)
++
++genrule(
++    name = "config_h",
```

Comment:
why isn't this just checked in as a source file?

Reply:
huh, you're right... there's a lot of cargo culting going on: https://github.com/tensorflow/tensorflow/blob/007eb8a5b793ecbaa648b2e894df4f4c846a6156/third_party/snappy.BUILD#L36